devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -58,17 +64,32 @@ metadata {
 }
 
 def updated() {
-    stopTimer()
-    startTimer()
+	log.debug "updated()"
+    if (device.getDataValue("manufacturer") == "SmartThings"){
+		stopTimer()
+        startTimer()
+    }
+    state.gsensor = 0
+    def thedetectTime = (detectTime ? detectTime as int : 2) * 1
+    def updatecmds = zigbee.writeAttribute(0x0000, 0x0000, 0x20, thedetectTime, [mfgCode: 0x120D]) 
+    log.debug "Updatecmds:  ${updatecmds}"
+	return response(updatecmds)
 }
 
 def installed() {
-    // Arrival sensors only goes OFFLINE when Hub is off
+	log.debug "installed()"
+	// Arrival sensors only goes OFFLINE when Hub is off
     sendEvent(name: "DeviceWatch-Enroll", value: JsonOutput.toJson([protocol: "zigbee", scheme:"untracked"]), displayed: false)
 }
 
 def configure() {
```

Comment:
Please, could You fix indentations in this method ? (and please change 4 spaces to 1 tab)

Reply:
OK! I will fix this indentation.